service/matching/taskQueueManager.go

```diff
@@ -292,6 +295,11 @@ func (c *taskQueueManagerImpl) AddTask(
 		c.liveness.markAlive(time.Now())
 	}
 
+	if params.forwardedFrom != "" {
+		// Only checks recent pollers in the root partition
```

Comment:
why only on root partition? Also this if condition is not accurate for checking root partition.

Reply:
My understanding is that on low usage case, having no recent pollers in a child partition is common, as most polls and add task calls will be forwarded to the rood partition for sync match. So:
By checking only the root partition,
1. If we do have a no poller issue, it will be detected at root partition.
2. If we do not have no poller issue, this avoids the FP detections in low usage case.
In contrast, If we check it for all partitions,
1. If we do have a no poller issue, it will be detected twice, once at original partition, once at root partition.
2.  If we do not have no poller issue, we have FP detections in low usage case.
Correct me if I'm wrong.

Reply:
that make sense.